packages/page/src/UserInterface/Controller/Admin/PageController.php

```diff
@@ -217,6 +219,20 @@ private function handleAction(Request $request, string $uuid): ?PageInterface //
             /** @see Sulu\Page\Application\MessageHandler\CopyLocalePageMessageHandler */
             /** @var null */
             return $this->handle(new Envelope($message, [new EnableFlushStamp()]));
+        } elseif ('move' === $action) {
+            $destinationUuid = $request->query->getString('destination');
+            $message = new MovePageMessage(['uuid' => $uuid], ['uuid' => $destinationUuid]);
+
+            /** @see Sulu\Page\Application\MessageHandler\MovePageMessageHandler */
+            /** @var null */
+            return $this->handle(new Envelope($message, [new EnableFlushStamp()]));
+        } elseif ('copy' == $action) {
+            $destinationUuid = $request->query->getString('destination');
+            $message = new CopyPageMessage(['uuid' => $uuid], ['uuid' => $destinationUuid]);
+
+            /** @see Sulu\Page\Application\MessageHandler\CopyPageMessageHandler */
+            /** @var null */
+            return $this->handle(new Envelope($message, [new EnableFlushStamp()]));
```

Comment:
return null sounds strange here. Copy an article ends on the copied version: https://sulu.rocks/admin/#/articles/en/article/95c9cbc7-cb46-4ad2-9b4f-ceb7ecb4ae76/details
So think we need response something here.